Gemfile

```diff
@@ -28,7 +28,7 @@ group :development do
   gem "puppet-module-win-dev-r#{minor_version}",       require: false, platforms: [:mswin, :mingw, :x64_mingw]
   gem "puppet-lint-i18n",                              require: false
   gem "net-telnet", '0.1.0',                           require: false if Gem::Version.new(RUBY_VERSION.dup) < Gem::Version.new('2.3.0')
-  gem "net-telnet",                                    require: false if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.3.0')
+  gem "net-telnet", '0.1.1',                           require: false if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2.3.0')
```

Comment:
also edit the .sync.yml or this will be lost. why the change ? does this clash with net-telnet in puppet agent ?

Reply:
.sync.yml still needs to be edited. Does this affect other modules ????

Reply:
Updated the sync file.
It conflicted with the specinfra gem as it pins net-telnet to 0.1.1

Reply:
Does this affect other modules ?

Reply:
Not that I've seen. inifile is the only one that seems to have net-telnet specifically set